yt/data_objects/time_series.py

```diff
@@ -56,16 +57,12 @@ def _eval(params, ds):
                 dict(eval = _eval, _params = tuple()))
     return cls
 
-def get_filenames_from_glob_pattern(filenames):
-    file_list = glob.glob(filenames)
-    if len(file_list) == 0:
-        data_dir = ytcfg.get("yt", "test_data_dir")
-        pattern = os.path.join(data_dir, filenames)
-        td_filenames = glob.glob(pattern)
-        if len(td_filenames) > 0:
-            file_list = td_filenames
-        else:
-            raise YTOutputNotIdentified(filenames, {})
+def get_filenames_from_glob_pattern(pattern):
```

Comment:
Well this function's name indicate that it's meant to be passed a pattern. It's true that, in practice, it receives the "outputs" param from `DatasetSeries.__new__()`, but only in as a special-case handling where the parameter doesn't actually hold "outputs" yet.

Reply:
If you’re not convinced I can change it, it’s not that crucial. Let me know !

Reply:
Ok, but the way I'm reading this there's no separation in functionality right? outputs will always be sent to this function. There's no separation if it's a list or a pattern. In __new__ we see `outputs = get_filenames_from_glob_pattern(outputs)`, so it isn't always a pattern.

Reply:
Actually, do we have a test verifying that a list works?

Reply:
It doesn't show up in the lines I've touched but there *is* a separation in `DatasetSeries.__new__`
https://github.com/yt-project/yt/blob/6befef2bc0427250fd62395962599be41b193e65/yt/data_objects/time_series.py#L141

I have no idea if we have a test for list. I considered adding one but I don't know of any sample data that'd be appropriate for this.

Reply:
I just verified that I could use a list with `enzo_tiny_cosmology`. 

That said, I know this is nitpicky but I think it's good to be consistent with how we send args in defintions vs. in the place they're used in our code. I would prefer if the definition also used `outputs` so we're consistent, even if there's an implication it will be a pattern of some sort.